tests/test_config.py

```diff
@@ -15,11 +16,23 @@
 
 
 def test_get_user_config_dir():
-    assert get_user_config_dir() == Path(__file__).parent / "data/user_dirs/config/openeo-python-client"
+    # For Linux/Mac we configure the config and data directories in conftest.py
+    # via the environment variables XDG_CONFIG_HOME and XDG_DATA_HOME.
+    # But on Windows we need to set APPDATA instead, and we don't have separate
+    # variables for config and data.
+    if platform.system() == "Windows":
+        os.environ["APPDATA"] = str(Path(__file__).parent / Path("data/user_dirs/config"))
+    assert get_user_config_dir() == Path(__file__).parent / Path("data/user_dirs/config/openeo-python-client")
 
 
 def test_get_user_data_dir():
-    assert get_user_data_dir() == Path(__file__).parent / "data/user_dirs/data/openeo-python-client"
+    # For Linux/Mac we configure the config and data directories in conftest.py
+    # via the environment variables XDG_CONFIG_HOME and XDG_DATA_HOME.
+    # But on Windows we need to set APPDATA instead, and we don't have separate
+    # variables for config and data.
+    if platform.system() == "Windows":
+        os.environ["APPDATA"] = str(Path(__file__).parent / Path("data/user_dirs/data"))
+    assert get_user_data_dir() == Path(__file__).parent / Path("data/user_dirs/data/openeo-python-client")
```

Comment:
just to make sure I understand how things work on windows:
that change of `"data/user_d...` to `Path("data/user_d..`   is that to make sure that dir separators are normalized to windows style?

Also: doesn't that just happen automatically because of the `/` operator in  `Path(...) / "...."`?

Reply:
That was indeed the why I initially wrapping that string in Path(...) 
However, I checked again and I was wrong. This is not necessary after all.  

Early on it seemed like it wasn't converting the string that was concatenated with the / operator. But I was mistaken and the error was something else.  So we don't actually need to wrap the second part in Path(...).

I have now removed the extra Path(...).
Simpler is better, less surprise is more readable.